crates/holochain_p2p/src/types/actor.rs

```diff
@@ -202,6 +202,7 @@ impl Default for GetActivityOptions {
 }
 
 /// Trait defining the main holochain_p2p interface.
+#[automock]
```

Comment:
Should this be behind the test-utils features?

Reply:
Oh yes! Done.